playkit/src/main/java/com/kaltura/playkit/PKRequestParams.java

```diff
@@ -27,5 +27,7 @@ public PKRequestParams(Uri url, Map<String, String> headers) {
          * @return The input object, or an adapted one.
          */
         PKRequestParams adapt(PKRequestParams requestParams);
+
+        void updateSessionId(String sessionId);
```

Comment:
Should be `updateParams(Player player)`. Other implementations may need other info.